modules/serverless-rn-1-6-0.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-6-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.6.0
+// add a version, e.g. Technology Preview 1.0.0
+
+[id="new-features-1-6-0_{context}"]
+== New features
+* {ServerlessProductName} 1.6.0 is available on {product-title} 4.3 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.13.1.
+* {ServerlessProductName} has been updated to use Knative `kn` CLI 0.13.1.
+* {ServerlessProductName} has been updated to use Knative Serving Operator 0.13.1.
+* The `serving.knative.dev` API group has now been fully deprecated, and is replaced by the `operator.knative.dev` API group.
++
+You must complete the steps described in the {ServerlessProductName} 1.4.0 release notes, related to updating from the `serving.knative.dev` API group to the `operator.knative.dev` API group, before you can upgrade to the latest version of {ServerlessProductName}.
++
+[IMPORTANT]
+====
+This change causes unspecified commands, such as `oc get knativeserving`, to become unreliable and not always work correctly.
```

Comment:
Will you please clarify "unspecified?" I don't think this is going to translate well.

Reply:
I think what we mean here is commands that don't have arguments / flags specified.
@markusthoemmes can you confirm what would be the best wording for what these actually are?

Reply:
Unspecified means: No fully qualified APIGroup + kind. `oc get knativeserving.operator.knative.dev` is fully qualified. `oc get knativeserving` isn't.